test/feature/spqr_test.go

```diff
@@ -209,6 +212,28 @@ func (tctx *testContext) connectPostgresqlWithCredentials(username string, passw
 	return db, nil
 }
 
+func (tctx *testContext) connectCoordinatorWithCredentials(username string, password string, addr string, timeout time.Duration) (*sqlx.DB, error) {
+	dsn := fmt.Sprintf("postgres://%s:%s@%s/%s", username, password, addr, dbName)
+	connCfg, _ := pgx.ParseConfig(dsn)
+	connCfg.DefaultQueryExecMode = pgx.QueryExecModeSimpleProtocol
+	connCfg.RuntimeParams["client_encoding"] = "UTF8"
+	connStr := stdlib.RegisterConnConfig(connCfg)
+	db, err := sqlx.Open("pgx", connStr)
+	if err != nil {
+		return nil, err
+	}
+	// sql is lazy in go, so we need ping db
+	testutil.Retry(func() bool {
+		_, err = db.Exec("SHOW routers")
```

Comment:
yes, maybe we need to develop coordinator-specific ping cmd, something like `SHOW STATUS` in coordinator, which checks coordinator installation and return health-check result

Reply:
for now `SHOW ROUTERS` is fine